checkov/terraform/checks/resource/aws/WAFHasLogs.py

```diff
@@ -0,0 +1,19 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.consts import ANY_VALUE
+
+class WAFHasLogs(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Logging is enabled for WAF  Web Access Control Lists"
+        id = "CKV_AWS_176"
+        supported_resources = ['aws_waf_web_acl', 'aws_wafregional_web_acl', 'aws_wafv2_web_acl']
```

Comment:
```suggestion
        supported_resources = ['aws_waf_web_acl', 'aws_wafregional_web_acl']
```
`aws_wafv2_web_acl` has a separate resource for logging, which you added via `CKV2_AWS_31` 😄